ai-platform-unified/notebooks/official/explainable_ai/gapic-custom_image_classification_online_explain.ipynb

```diff
@@ -31,7 +31,7 @@
     "repo": "snippets_housekeeping.ipynb"
```

Comment:
<p>Seems like it's called the "Vertex SDK for Python" now, if you're talking about the higher level library. Otherwise, if you're talking about the GAPIC one, I think it is the Vertex AI client library for Python</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>My understanding is we only use the "AI" in "Vertex" when talking about the platform itself. Otherwise, if we are talking about a product/service, we drop AI from the branded name.</p>

Reply:
The GAPIC library is considered to be part of the Vertex SDK for Python. We need a consistent way to refer to the GAPIC library. Might be worth a conversation with Sasha about this.

Reply:
Please use "Vertex AI" and "Vertex Explainable AI" wherever appropriate